cmd/root.go

```diff
@@ -70,27 +73,35 @@ Tarballs can also be specified by simply providing the path to the .tar, .tar.gz
 }
 
 func outputResults(resultMap map[string]util.Result) {
+
 	// Outputs diff/analysis results in alphabetical order by analyzer name
 	sortedTypes := []string{}
 	for analyzerType := range resultMap {
 		sortedTypes = append(sortedTypes, analyzerType)
 	}
 	sort.Strings(sortedTypes)
 
+	// Get the writer
+	writer, err := getWriter(outputFile)
+	if err != nil {
+		logrus.Error(err)
```

Comment:
can you wrap the error here for a bit more context? `errors.Wrap(err, "getting writer for file")` or something like that